pkg/types/defaults/installconfig.go

```diff
@@ -44,7 +44,7 @@ func SetInstallConfigDefaults(c *types.InstallConfig) {
 		}
 	}
 	if c.Networking.NetworkType == "" {
-		if c.IsOKD() {
+		if c.IsOKD() || c.IsSingleNodeOpenShift() {
 			c.Networking.NetworkType = defaultOKDNetworkType
```

Comment:
Sorry in advance to be nitpicky here, and I'm only leaving this comment because there was another change in this review. Also, the issue I'm complaining about here does not really come from this PR, but I think we should fix it:

I think the semantics here are bad with the variable `defaultOKDNetworkType` and it bothers me a little bit to assign the `defaultOKDNetworkType` to SNO, because I don't see a relationship between the networking type of OKD and SNO. I think we should just get rid of the `defaultOKDNetworkType` variable (I don't think it is widely used) and then:

```suggestion
		if c.IsOKD() || c.IsSingleNodeOpenShift() {
			c.Networking.NetworkType = string(operv1.NetworkTypeOVNKubernetes)
```

Reply:
Agree. I was reusing that variable and didn't pay too much attention to its name. It would be confusing to keep using the same variable name.